docs/server-config.md

```diff
@@ -10,3 +10,4 @@ environment variables. You can specify the configuration file to use with the `-
 | --- | --- | --- |
 | `port` | `Integer` | Port to bind the API server to, overridden by env `APP_SERVER_PORT`. |
 | `dev` | `Boolean` | If set to true the server runs in development mode serving also static content. |
+| `dockerWindows` | `Boolean` | Set this to true if you're using Docker for Windows (Only Win10 Pro/Education). If you're using Docker Toolbox (other Windows versions), leave this to false. Default is false. |
```

Comment:
Could this be autodetected?

Reply:
Maybe, but let's do it in some other PR preferrably